packages/backend/src/ee/controllers/aiAgentController.ts

```diff
@@ -265,6 +267,64 @@ export class AiAgentController extends BaseController {
         };
     }
 
+    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
+    @SuccessResponse('200', 'Success')
+    @Get('/projects/{projectUuid}/conversations')
+    @OperationId('getAiAgentConversations')
+    async getAiAgentConversations(
+        @Request() req: express.Request,
+        @Path() projectUuid: string,
+    ): Promise<ApiAiConversations> {
+        this.setStatus(200);
+        return {
+            status: 'ok',
+            results: await this.getAiAgentService().getConversations(
+                req.user!,
+                projectUuid,
+            ),
+        };
+    }
+
+    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
+    @SuccessResponse('200', 'Success')
+    @Get('/projects/{projectUuid}/conversations/:aiThreadUuid/messages')
+    @OperationId('getAiAgentConversationMessages')
+    async getAiAgentConversationMessages(
+        @Request() req: express.Request,
+        @Path() projectUuid: string,
+        @Path() aiThreadUuid: string,
+    ): Promise<ApiAiConversationMessages> {
+        this.setStatus(200);
+        return {
+            status: 'ok',
+            results: await this.getAiAgentService().getConversationMessages(
+                req.user!,
+                projectUuid,
+                aiThreadUuid,
+            ),
+        };
+    }
```

Comment:
`conversations` vs `threads` is becoming confusing; I assume these will be deprecated soon